### PR TITLE
Fix wrong statements about Meilisearch

### PR DESCRIPTION
**Pull request by Kerollmops**
Hey 👋,

Happy to propose this set of modifications. Could you take a look at it, please? I am happy to answer your questions and help to decide about anything by providing links, documentation, or anything else.

## Change Summary
In this PR, I propose fixing the wrong statements about Meilisearch. As we worked a lot, we improved the engine and reduced most of the limitations.

## PR Checklist
- [x] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: typesense.org/pages/typesense-vs-algolia-vs-elasticsearch-vs-meilisearch.vue [hunk 1]
```diff
@@ -632,15 +661,8 @@
...
                   </td>
-                  <td class="text-danger">
-                    Slow indexing performance
-                    <a
-                      href="https://github.com/meilisearch/meilisearch/discussions/2131"
-                      target="_blank"
-                      class="text-danger"
-                      >reported</a
-                    >
-                    for over 10K records.
```
Comment (Kerollmops) [inline typesense.org/pages/typesense-vs-algolia-vs-elasticsearch-vs-meilisearch.vue:643]: Here is one of the many [users reporting](https://github.com/meilisearch/meilisearch/discussions/3450#discussioncomment-4862004) that he is using Meilisearch for 3x to 4x more than the limit you are talking about that is one year old.

**Comment by jasonbosco** [comment]
Thank you for the updates. I'll have some edits to this, but I'll do those after merging. 

In the meantime, reminder about these changes: https://twitter.com/typesense/status/1620825236055932928?s=20

**Comment by Kerollmops** [comment]
> In the meantime, reminder about these changes: https://twitter.com/typesense/status/1620825236055932928?s=20

Could you please [do a PR](https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/comparison_to_alternatives.md) as I did on your comparison page?

**Comment by jasonbosco** [comment]
Yup, just opened this PR: https://github.com/meilisearch/documentation/pull/2174

**Pull request merged**